tests/conftest.py

```diff
@@ -149,12 +149,19 @@ def calc_date_params(backdate, setdate):
     return acceptance_date
 
 
+@pytest.fixture
+def initialized_with_research_study(initialized_db):
```

Comment:
Would it make sense for this fixture to return a reference (SQLA queryset?) to the data that's been added, so tests that depend on the fixture can introspect the fixture?

Reply:
complicated :)  that one is referencing a known static value - but we do return live objects where they're useful to the tests.